splunk_connect_for_snmp_poller/manager/tasks.py

```diff
@@ -72,98 +147,23 @@ def snmp_polling(
-                                )
-                        else:
-                            # Perform SNMP polling for oid
-                            try:
-                                if varbind[-1] == "*":
-                                    walk_handler(
-                                        snmp_engine,
-                                        auth_data,
-                                        context_data,
-                                        host,
-                                        port,
-                                        varbind,
-                                        mib_server_url,
-                                        index,
-                                        otel_logs_url,
-                                        otel_metrics_url,
-                                        one_time_flag,
-                                    )
-                                else:
-                                    get_handler(
-                                        snmp_engine,
-                                        auth_data,
-                                        context_data,
-                                        host,
-                                        port,
-                                        varbind,
-                                        mib_server_url,
-                                        index,
-                                        otel_logs_url,
-                                        otel_metrics_url,
-                                        one_time_flag,
-                                    )
-                            except Exception as e:
-                                logger.error(
-                                    f"Invalid format for oid. Error message: {e}"
-                                )
+                # Divide varBinds for WALK/BULK actions
+                varbind_collection = sort_varbinds(varBinds)
+                logger.info(f"Varbind collection: {varbind_collection}")
+                # Perform SNMP BULK
+                get_data(varbind_collection.bulk, bulk_handler, *static_parameters)
+                # Perform SNMP WALK
+                get_data(varbind_collection.get, get_handler, *static_parameters)
         # Perform SNNP Polling for oid profile in inventory.csv
         else:
             # Perform SNNP WALK for oid end with *
             if profile[-1] == "*":
                 logger.info(f"Executing SNMP WALK for {host} profile={profile}")
-                walk_handler(
-                    snmp_engine,
-                    auth_data,
-                    context_data,
-                    host,
-                    port,
-                    profile,
-                    mib_server_url,
-                    index,
-                    otel_logs_url,
-                    otel_metrics_url,
-                    one_time_flag,
-                )
+                walk_handler(profile, *static_parameters)
```

Comment:
I'd rename walk_handler in snmp_walk_handler, and get_handler in snmp_get_handler